list-ops/list_ops_test.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import math
+
+from list_ops import map_src, length_src, reduce_src
+
+
+class ListOpsTest(unittest.TestCase):
+
+    # necessary functions to test map_src
+    def square(self, x):
+        return x**2
```

Comment:
I agree, but isn't it slightly confusing to reference the original map function in the tests since I changed the function name to map from map_src?